src/card/card.js

```diff
@@ -23,7 +23,11 @@ export default (data, cardStyling, selectionObj) => {
   const topColor = colorUtils.getDarkColor(backgroundColor);
   const fontColor = getFontColor(cardStyling, backgroundColor);
   const attributes = data.attributes || {};
-  let html = `<div class="sn-org-card-title">${attributes.label || data.id}</div>`;
+  let html = '<div class="sn-org-card-content">';
+  if (attributes.imageURL) {
+    html += `<img class="sn-org-card-image" src="${attributes.imageURL}"/>`;
+  }
+  html += `<div class="sn-org-card-textarea"><div class="sn-org-card-title">${attributes.label || data.id}</div>`;
```

Comment:
hmm, possible that these also should be escaped?
`='<script type="text/javascript">function() {alert.show("hey")}()</script>'`
Doesn't really do anything in the client (chrome) but feels a bit risky as I can see it inject the node.

Reply:
can't get it to run any of my injected code, so might be fine. But still should probable add a html encoding to all of these.